src/Composer/ScriptHandler.php

```diff
@@ -29,6 +29,8 @@ class ScriptHandler
      */
     public static function initializeApplication(Event $event)
     {
+        static::purgeCacheFolder();
```

Comment:
You don't initialize the app, just add the folder for it. Imo the order does not matter at all. Also the cache folder and app folder are not connected in any way.